src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const male = people.filter(person => {
+    return century
```

Comment:
You need to use more reliable check here because century theoretically can have valid falsy value (0)
```suggestion
    return century !== undefined
```